### DISCUSSION TITLE
Paid Job - I apologize if posted in wrong area.

### DISCUSSION THREAD

**Discussion by OldeSchoolWolf** [kevinzg/facebook-scraper#173] [General]
I need my Facebook group, all posts and comments, scrapped. I do not have the leisure of time to figure this out on my own as FB has begun purging alternative health groups. Please respond and we can discuss details and pay.

**Comment by maroxtn** [comment]
Contact me: boukil98@gmail.com

**Comment by chowdhury-najir** [comment]
You can contact me,at najir.cse.sust@gmail.com
I have already done  similar kind of project recently.
